### DISCUSSION TITLE
Cfclient cannot detect crazyflie

### DISCUSSION THREAD

**Discussion by francesco2706** [bitcraze/discussions#1610] [Q&A] [answered]
hi, I'm a new crazyflie user and I can't connect the drone in the cfclient. I'm on Windows and in particular, after inserting the crazyradio PA USB dongle and checking in device management that it is actually detected, I open the cfclient and turn on the crazyflie. By clicking on scan I am unable to find any crazyflies. Even if I connect the crazyflie to the USB I cannot configure it manually, as I want to underline that the drones I am using, given that they are in a laboratory, have been previously used by other people.

![Screenshot 2024-10-28 195009](https://github.com/user-attachments/assets/c8bd4657-c583-44d0-911d-1a0fc336a609)

**Comment by ataffanel** [comment]
Have you installed the libusb USB driver with Zadig? This is a required step to get the radio to work on Windows, it is documented in the getting started documentation.

If the radio is well installed, you should see it blinking RED when you launch the client and when clicking the scan button, if it is not the case, the radio driver is likely not installed properly.

Ps. Please avoid spamming the discussion. I empathized that you might be stressed but it is bad etiquette to spam multiple threads and to expect hourly answer on this forum. In doubt please see the discussion rules: https://github.com/orgs/bitcraze/discussions/55.

**Reply by francesco2706** [reply]
Yes, I do. The radio blinks red when I click scan or when I lunch the cfclient.

**Comment by knmcguire** [comment]
Hi! Could you perhaps also share how the LEDs look like on the Crazyflie. Perhaps it is bricked and that is why the cfclient can't connect to it.

The crazyflie in usb needs to also be configured by Zadig same way as the crazyradio in order to connect to it through USB

**Reply by francesco2706** [reply, marked as answer]
I solved it, I hadn't used zadig for the crazyflie. Thank you very much for your availability
